docs/manual/ja/introduction/How-to-update-things.html

```diff
@@ -149,6 +149,7 @@ <h2>Materials</h2>
                     <li>morphing</li>
                     <li>shadow map</li>
                     <li>alpha test</li>
+										<li>transparent</li>
```

Comment:
Tabs vs. spaces 😱

Reply:
This clean up can be done in a separate PR. It necessary in the entire `jp` directory.